.github/workflows/docs-site.yml

```diff
@@ -0,0 +1,59 @@
+name: Docs Site Deploy
+
+on:
+  push:
+    branches: [ main ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
```

Comment:
Is anything else required under this section or that's all thats required if we want to run the workflow manually?

Reply:
Just having this section present, should allow manual invocation